build.gradle

```diff
@@ -269,7 +269,13 @@ dependencyAnalysis {
         }
         project(":vector") {
             onUnusedDependencies {
-                exclude("org.maplibre.gl:android-sdk", "org.maplibre.gl:android-plugin-annotation-v9") // False positives
+                // False positives
+                exclude(
+                        "com.vanniktech:emoji-google",
+                        "com.vanniktech:emoji-material",
+                        "org.maplibre.gl:android-sdk",
+                        "org.maplibre.gl:android-plugin-annotation-v9",
```

Comment:
Sorry to be so painful, but `p` < `s` :)

Reply:
ah I only checked emoji deps :facepalm:

Reply:
thanks!